app/src/util.js

```diff
@@ -243,7 +243,26 @@
                     // TODO report error
                 }, this));
             }
+        },
+
+        // Display a bootstrap-style alert message to the user.
+        // Type should be success, info, warning, or danger.
+        // Timeout is how long the alert should be display.  Defaults to 5 seconds.
+        bootstrapAlert: function (type, message, timeout) {
+            timeout = typeof timeout !== 'undefined' ? timeout : 5;
+            timeout *= 1000; // convert to milliseconds
+
+            $('#alert_placeholder').html('<div id="alert" class="alert alert-' + type + ' alert-dismissable fade in"><button type="button" class="close" data-dismiss="alert" aria-hidden="true">&times;</button><span>' + message + '</span></div>');
```

Comment:
Thanks for the feedback.  I just pushed a new commit that more carefully handles the message parameter.

I agree that inlining HTML isn't ideal.  My goal here was to replicate Javascript's built-in alert() function using bootstrap's prettier alert classes.  Bootstrap's alerts are removed from the DOM when they are closed though, so I don't think we could get away with adding this bit of HTML to the Jade template.